mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModel.kt

```diff
@@ -529,7 +532,8 @@ abstract class MavericksViewModel<S : MvRxState>(
             flowWhenStarted(lifecycleOwner)
         }
         val scope = lifecycleOwner?.lifecycleScope ?: viewModelScope
-        return scope.launch {
+        return scope.launch(start = CoroutineStart.UNDISPATCHED) {
```

Comment:
@denis-bezrukov Can you add a comment here explaining it like in your pr description?